Two item meta info

SebastianGerS opened the pull request:
I made this pull request mostly to get some feedback.
I added the image to, just to see how it would look so basically this is the entire page for a single item
Ofcourse some logic for when seasons are displayed or not will be added when we start to use the db info
not that I have not done any adaptions for smaller screens yet.

Commit:
create basic setup for working for creating component

Commit:
added files to .gitignore

Commit:
solve merge conflict corecct file (folder) in .gitignore

Commit:
solve mergeconflict

Commit:
create some basic html markup

Commit:
fix mergeconflict bringin in new varibles

Commit:
create markup and basic layout for item-meta-info

	* add som basic styling
	* add markup for tables
	* add new varibles for scss

Commit:
fix mergeconflict

Commit:
fix correct paths to files after moving files to new directories

Commit:
add material to html file and adjust layout

	* add new section for plot summary and add list of actors to
	  existing section
	* add img
	* move item-header to its own div

Commit:
add aditional div to center entire content

AnnaRoss commented on resources/assets/sass/components/item_meta_info.scss:
```diff
@@ -0,0 +1,162 @@
+}
+
+.item-meta-info {
+
```
An empty line has slipped in here :smile:

SebastianGerS replied:
should be fixed now, also restructured it a bit now always leaving one blank row before writing a property:value pair for a given selector

SebastianGerS replied:
but with html I realy like it when there's no blank rows at all for some wierd reasson, but I get your point it's probably more readable with some blank rows

AnnaRoss commented on resources/assets/sass/components/item_meta_info.scss:
```diff
@@ -0,0 +1,162 @@
+    justify-content: space-between;
+    font-size: $sub-heading-font-size;
+    color: $heading-alt-color;
+
```
An empty line has slipped in here 😄

AnnaRoss commented on resources/views/layouts/item_meta_info.blade.php:
```diff
@@ -1,4 +1,27 @@
+      <!-- Styles -->
+      <link href="{{asset('css/item_meta_info.css')}}" rel="stylesheet">
+   </head>
+  </body>
```
This should be an opening body tag instead?

AnnaRoss commented on resources/views/layouts/components/item_meta_info.blade.php:
```diff
@@ -12,83 +12,85 @@
+              </table>
+            </div>
+            <div class="row-flex-start">
+              <h3>Short</h3><h2>Facts</h2>
```
I think it's a bit of a misuse to use `<h3>` and `<h2>` elements in this way. Maybe we could just make it :
```html
<h2>short<strong>Facts</strong></h2>
```
That isn't very semantically correct either. Just a thought. 

Or perhaps that we make use of the `<caption>` tag:
```html
<table>
      <caption>Short Facts</caption>
</table>
```
Or just some more lovely `<span>` elements. 
We could also just let go of how it looks in the prototype.

SebastianGerS replied:
I changed it to 
`<h2><span>Short</span><span>Facts</span></h2>`

and regarding the other semantic stuff I changed some of the div to other types of elements, seams to work fine : ) 
Hard to determine how to correctly use sections and articles thought, now I have some sections in sections and stuff like that....

AnnaRoss reviewed (approved):
Some general comments: 

* Maybe you could insert empty lines between significant elements in the item_meta_info.blad.php. For example:
```
<body>

     <div></div>
```
```
</table>

<div class="plot-summary">
```
If it is possbile to exchange some of the `<div>` tags into `<section>` it would be great - thinking about a11y and _search engine optimisation_. But I don't know if it would affect the current css too much - it is not a priority! Most importantly - it works.

Commit:
Semantic fixes

	* use span inside a h2 instead of two diffrent hedding tags
	* replace some divs with main,article,section
	* add new varible for biger paragraph font

Commit:
fix corecct spacing

Commit:
Fix responsive layout

	* add @media rules for smal and medium sized screens

Commit:
fix corect url to fonts

Commit:
remove blank rows

Commit:
fix spacing
